desktop-exporter/trace_data.go

```diff
@@ -0,0 +1,40 @@
+package desktopexporter
+
+import (
+	"time"
+)
+
+func (trace *TraceData) GetTraceSummary() (*TraceSummary, error) {
```

Comment:
`TraceSummary` doesn't need to be a pointer here

https://www.reddit.com/r/golang/comments/r92m5k/best_practices_with_pointers/hnfk2x6/
https://medium.com/@meeusdylan/when-to-use-pointers-in-go-44c15fe04eac
https://golongwithgolang.com/when-should-you-use-pointers-in-go

```suggestion
func (trace *TraceData) GetTraceSummary() (TraceSummary, error) {
```